pkg/pillar/types/zedroutertypes.go

```diff
@@ -1583,7 +1684,7 @@ func GetExistingInterfaceList(log *base.LogObject, globalStatus DeviceNetworkSta
 	return ifs
 }
 
-// Check if an interface name is a port owned by zedrouter
+// Check if an interface name is a port owned by nim/zedrouter
```

Comment:
Actually the original zedrouter is just wrong; owned by nim is the correct one AFAIK

Reply:
Fixed